server/utils/db/db_utils.py

```
@@ -1598,6 +1598,22 @@ def initialize_tables():
 
+            # Migration: Add change_gating_enabled to connected_repos so
+            # existing deployments can enroll repos in PR change gating.
+            try:
+                cursor.execute(
+                    "ALTER TABLE connected_repos ADD COLUMN IF NOT EXISTS change_gating_enabled BOOLEAN DEFAULT FALSE;"
+                )
+                conn.commit()
+                logging.info(
+                    "Ensured change_gating_enabled column exists on connected_repos table."
+                )
+            except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)